### ISSUE TITLE
App screen is blank

### ISSUE THREAD

**Issue by archee** [cowbell/ionic-geofence#10]
Running the app with `ionic run android` installs the app onto my device (Samsung Galaxy S6) but when the app opens, a blank white screen is displayed with no ability to interact with the app. The same occurs when running in an Android emulator using target API level 19.

**Comment by tsubik** [comment]
Hey @archee I've updated also name of the module while [loading it here](https://github.com/cowbell/cordova-plugin-geofence/commit/84a784af57a3f06230f1e7b2b3ca381f31b04de0). It should work fine now.

**Comment by archee** [comment]
Hi @tsubik 

Did a fresh clone of the example app and I'm still seeing the same behavior. I did a `ionic serve` to view the developer console and noticed a bunch of 404's on what seems to be needed asset files. I also made sure to the `npm install` beforehand just in case.

Here is a screenshot of my developer console when running `ionic serve` of this example app.
![screen shot 2015-07-07 at 3 58 12 pm](https://cloud.githubusercontent.com/assets/1059318/8557164/2a341b96-24c1-11e5-887c-1b39f35c0175.png)
